pkgs/applications/networking/circumflex/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, less, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "circumflex";
+  version = "2.6";
+
+  src = fetchFromGitHub {
+    repo = "circumflex";
+    owner = "bensadeh";
+    rev = "2.6";
```

Comment:
Maybe use `version` here just to have one place to update later?

Reply:
can you elaborate further? You mean version as an argument to `fetchFromGithub`? what should it contain? and where can I read more about it if possible?

Reply:
I mean using `rev = version;`, so you get the `version` value from the outer attrset.

Reply:
I see, makes sense. I changed it as requested.

Reply:
This is a strong requirement and should be done everywhere possible.